README.md

```diff
@@ -62,6 +62,10 @@ is to consider anything connecting to the bus specified when starting dbus-proxy
 as being on the 'inside' and thus any interaction with the bus specified when
 starting dbus-proxy is 'outgoing'.
 
+When the proxy is interfered with D-Bus message, configuration list filtered due to
```

Comment:
interfered is the wrong word here, intercepts is probably closer to what we want? Also the configuration list _is_ filtered.